File: FrontEnd/src/app/models/message.ts
export class Message {

  content: string;
  timestamp: string;
  is_it_mine: boolean;
  otherUsername: string;

  constructor(content: string, timestamp: string, is_it_mine: boolean, otherUsername: string) {
    this.content = content;
    this.timestamp = timestamp;
    this.is_it_mine = is_it_mine;
    this.otherUsername = otherUsername;
  }
}

File: BackEnd/src/main/java/ma/fstt/backend/Message/Message.java
package ma.fstt.backend.Message;


import jakarta.persistence.*;
import lombok.*;

@Data
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "message")
public class Message {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String content;
    private String timestamp;
    private String Sender;
    private String Receiver;

}
